convention.md

```diff
@@ -18,11 +18,49 @@ The underscore (`_`) is used and reserved for Homie *node arrays*.
 
 ### Payload
 
-Every MQTT message payload MUST be sent as UTF-8 encoded string.
-A numeric data type MUST be converted to a literal string with the dot character as the decimal separator for the `float` data type.
-An empty message is not valid as payload for numeric data types.
-The value published as payload MUST be valid for the respective property/attribute type as per the convention definition below.
-Booleans MUST be converted to "true" or "false".
+ - Every MQTT message payload MUST be sent as a UTF-8 encoded string
+-  The value published as payload MUST be valid for the respective property/attribute type as per the defined convention
+ 
+String
+ - String types are limited to 268,435,456 characters 
+ - An empty string ("") is a valid payload
+ 
+Integer
+ - Integer types are UTF-8 encoded representations of 64-bit signed whole numbers
```

Comment:
The previous state was:

> A numeric data type MUST be converted to a literal string with the dot character as the decimal separator for the `float` data type.

The first part of this is not equally good represented here. "UTF-8 encoded representations" does not imply "converted to a literal string"

Reply:
Fixed